CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Get binary architectures use `lipo -archs`.  
```

Comment:
can you please add a link to the pull request?

Reply:
ok